site/examples/richtext.tsx

```diff
@@ -22,11 +22,20 @@ const HOTKEYS = {
 const LIST_TYPES = ['numbered-list', 'bulleted-list']
 const TEXT_ALIGN_TYPES = ['left', 'center', 'right', 'justify']
 
-const RichTextExample = () => {
-  const [value, setValue] = useState<Descendant[]>(initialValue)
+const RichTextExample = ({
```

Comment:
Change needed to make editor components test friendly. See https://github.com/mwood23/slate-test-utils#making-your-editor-test-friendly for more details on why